src/App.js

```diff
@@ -1,28 +1,55 @@
 import React, { useState } from 'react';
-import './App.css';
 import ReactPolling from 'react-polling'; 
+import DailyForecastCard from './Components/DailyForecastCard';
+import ForecastTitle from './Components/ForecastTitle';
+import HeatIndicator from './Components/HeatIndicator';
+import WeatherCondition from './Components/WeatherCondition';
+import WeatherHumidity from './Components/WeatherHumidity';
+import WeatherTemp from './Components/WeatherTemp';
+import WeatherDate from "./Components/WeatherDate";
+import ForecastList from './Components/ForecastList';
+import AppHeader from "./Components/AppHeader";
+import AppFooter from './Components/AppFooter';
+import WeatherIcon from "./Components/WeatherIcon";
 
 function App() {
   const apiKey = process.env.REACT_APP_WEATHER_API_KEY
   const lat =  "-23.5489";
   const lon =  "-46.6388";
   const part = "hourly,minutely";
+  const url = `https://api.openweathermap.org/data/2.5/onecall?lat=${lat}&lon=${lon}&exclude=${part}&appid=${apiKey}&units=metric`
   const [weatherData, setWeatherData] = useState([])
 
   const convertTimestampToDate = (unixTimestamp) => {
     let date = new Date(unixTimestamp * 1000);
     let day = date.getDate(); 
     let month = date.getMonth() + 1;
-    let year = date.getFullYear();
+    /* let year = date.getFullYear(); */
```

Comment:
Are we using this `year` variable yet? If not, I think we can remove this comment.

Reply:
No, we are not. Forgot to remove the comment.
I decided to remove the year because the App only shows 5-day forecast, starting from the current day.